tests/src/CLRTest.Execute.Batch.targets

```diff
@@ -229,9 +229,17 @@ IF NOT "%CLRTestExitCode%"=="%CLRTestExpectedExitCode%" (
 
     <PropertyGroup> 
       <_CLRTestRunFile Condition="'$(CLRTestIsHosted)'=='true'">"%CORE_ROOT%\corerun.exe"</_CLRTestRunFile>
+      <BatchCLRTestLaunchCmds><![CDATA[
+IF NOT "%CLRCustomTestLauncher%"=="" (
+  set LANUCHER=call %CLRCustomTestLauncher% %~dp0
+) ELSE (
+  set LANUCHER=%_DebuggerFullPath%
+)
+      ]]></BatchCLRTestLaunchCmds>
       <BatchCLRTestLaunchCmds Condition=" '$(IlasmRoundTrip)'=='true' "><![CDATA[
-ECHO $(_CLRTestRunFile) $(TargetAssemblyName) %CLRTestExecutionArguments% %Host_Args%
-$(_CLRTestRunFile) $(TargetAssemblyName) %CLRTestExecutionArguments% %Host_Args%
+$(BatchCLRTestLaunchCmds)
+ECHO %LAUNCHER% $(_CLRTestRunFile) $(TargetAssemblyName) %CLRTestExecutionArguments% %Host_Args%
```

Comment:
$(_CLRTestRunFile) is set to corerun.exe, which means this generates something like

```
%LAUNCHER% "%CORE_ROOT%\corerun.exe" Finalizer.exe %CLRTestExecutionArguments% %Host_Args%
```

When it should generate
```
%LAUNCHER% Finalizer.exe %CLRTestExecutionArguments% %Host_Args%
```